vendor_prop.mk

```diff
@@ -38,7 +38,7 @@ PRODUCT_PROPERTY_OVERRIDES += \
 
  # Bluetooth
 PRODUCT_PROPERTY_OVERRIDES += \
-    persist.bt.max.hs.connections=2 \
+    persist.bt.max.hs.connections=5 \
```

Comment:
Do-able but our stock sets persist.bt.max.hs.connections to 2 only, and that's what we were following.